examples/StandAloneListener/standalonelistener.py

```diff
@@ -1,11 +1,38 @@
 '''
-        This python script will listen to the defined vip address for specific
-    topics.  The user can modify the settings.py file to set the specific
-    topics to listen to.
 
-    With a volttron activated shell this script can be run like:
+    1. Make sure volttron instance is running using tcp address. use vcfg
+    command to configure volttron instance address,.
 
-       python standalonelistener.py
+    2. Update settings.py
+
+    3. Add this standalone agent to volttron auth entry using vctl auth add
+    command. Provide ip of the volttron instance when prompted for address[]:
+    and  provide public key of standalone agent when prompted for
+    credentials[]:
+    For more details see
+    https://volttron.readthedocs.io/en/develop/devguides/walkthroughs/Agent-Authentication-Walkthrough.html
+
+    Example command:
+
+    (volttron)[vdev@cs_cbox myvolttron]$ vctl auth add
+    domain []:
+    address []: 127.0.0.1
+    user_id []:
+    capabilities (delimit multiple entries with comma) []:
+    roles (delimit multiple entries with comma) []:
+    groups (delimit multiple entries with comma) []:
+    mechanism [CURVE]:
+    credentials []: GsEq7mIsU6mJ31TN44lQJeGwkJlb6_zbWgRxVo2gUUU
+    comments []:
+    enabled [True]:
+
+    4. With a volttron activated shell this script can be run like:
+
+       python test_weather2.py
```

Comment:
Shouldn't this use standalonelistener.py not your weather agent?

Reply:
sorry fixed it.